test/types/methods.types.ts

```diff
@@ -155,6 +155,7 @@ const columnSummary: Handsontable.plugins.ColumnSummary = hot.getPlugin('columnS
 const comments: Handsontable.plugins.Comments = hot.getPlugin('comments');
 const contextMenu: Handsontable.plugins.ContextMenu = hot.getPlugin('contextMenu');
 const copyPaste: Handsontable.plugins.CopyPaste = hot.getPlugin('copyPaste');
+const customBorders: Handsontable.plugins.customBorders = hot.getPlugin('customBorders');
```

Comment:
This error `Namespace 'Handsontable.plugins' has no exported member 'customBorders'.` shows that we need use `Handsontable.plugins.CustomBorders` (capital letter `CustomBorders`).

Reply:
True. I've added the capital letter in the next commit.